@here/harp-datasource-protocol/test/StyleSetEvaluatorTest.ts

```diff
@@ -879,7 +879,8 @@ describe("StyleSetEvaluator", function () {
             assert.strictEqual(techniques.length, 0);
         });
     });
-    it("lookup expressions and lookup table are cached", function () {
+    //not supported at the moment, see HARP-17877
+    it.skip("lookup expressions and lookup table are cached", function () {
```

Comment:
I'd propose to enable the test back and comment out only `expect(lookupSpy.callCount).equals(entries.length);` and write a comment about HARP-17877 above that line. Because this test also checks that the lookup table, that has been converted from an Array to Map on the first call, is reused again and that it is actually a Map instance.

Reply:
done